admin_guide/overcommit.adoc

```diff
@@ -212,29 +212,28 @@ You can save your definition to a file, for example *_resource-reserver.yaml_*,
 then place the file in the node configuration directory, for example
 *_/etc/origin/node/_* or the `--config=<dir>` location if otherwise specified.
 +
-At the same time node server needs to be configured to read the definition from the
-node configuration directory. If *_/etc/origin/node/_* is specifed, you can set the
-following in the
+Additionally, the node server needs to be configured to read
+the definition from the node configuration directory,
+by naming the directory in the `kubeletArguments.config` field of the
 link:../install_config/master_node_configuration.html[node configuration file]
-(the *_node-config.yaml_* file):
+(usually named *_node-config.yaml_*):
 +
 ====
 ----
 kubeletArguments:
   config:
-    - "/etc/origin/node"
+    - "/etc/origin/node"  <1>
 ----
+<1> If `--config=<dir>` is specified, use `<dir>` here.
 ====
 +
-With the *_resource-reserver.yaml_* file in place, on start of the node server,
-the node agent launches the specified container,
-and the remaining capacity for the scheduler to place cluster pods adjusts accordingly.
+With the *_resource-reserver.yaml_* file in place,
+starting the node server also launches the `sleep-forever` container.
```

Comment:
Should sleep-forever be placed in asterisks (bold) instead of back-ticks since it is a name of a container and not an inline command?